### PR TITLE
Add caption for gmeyer39

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @rogeryanda
- [ ] @jmedinacsu
- [ ] @js3263827
- [ ] @kayakkat
- [ ] @rjenkins5725
- [ ] @liam09
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @IAmHughes
- [ ] @edelgadojr
- [ ] @Phat6
- [ ] @octowilliam
- [ ] @jrhixson2002
- [ ] @dannyreid
- [ ] @mgthompson2811
- [ ] @anshubhatia
- [ ] @rogeryanda
- [ ] @githubteacher
- [ ] @Phat6
- [ ] @dannyreid
- [ ] @jrhixson2002
- [ ] @js3263827
- [ ] @jmedinacsu
- [ ] @IAmHughes
- [ ] @liam09
- [ ] @rjenkins5725
- [ ] @edelgadojr
- [ ] @rogeryanda
- [ ] @octowilliam
- [ ] @TJSaint
- [ ] @SpringsLW
- [ ] @rtsinclair
- [ ] @rogeryanda
- [ ] @gmeyer39
- [ ] @rtsinclair
- [ ] @lkelly1
- [ ] @js3263827
- [ ] @githubteacher
- [ ] @githubteacher
- [ ] @kayakkat
- [ ] @jrhixson2002
- [ ] @jrhixson2002
- [ ] @gmeyer39
- [ ] @lgibbens

### PR DESCRIPTION
**Pull request by gmeyer39**
TL;DR - One sentence of what's going on.

## Tasks remaining
- [x] Get a review

Link to issue or service now
This resolves#2

Lead questions

### CODE REVIEW

File: _posts/0000-01-44-gmeyer39.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (lkelly1) [inline _posts/0000-01-44-gmeyer39.md:4]: Jeff can trespass

**Review by lkelly1 — APPROVED** [review]
Fine, trespass - no trespass - whatever.
